libtiledbvcf/src/stats/variant_stats_reader.h

```diff
@@ -102,6 +103,46 @@ class AFMap {
            pos_map.first;
   }
 
+  /**
+   * @brief Compute the Allele Frequency for an allele at the given position,
+   * accounting for the full population of the dataset
+   *
+   * @param pos Position of the allele
+   * @param allele Allele value
+   * @param num_samples Number of samples in the entire dataset
+   * @return float Allele Frequency
+   */
+  float af(
+      uint32_t pos,
+      const std::string& allele,
+      std::vector<std::vector<char>>::size_type num_samples) {
+    // calculate af = ac / an
+    std::unordered_map<
+        uint32_t,
+        std::pair<int, std::unordered_map<std::string, int>>>::const_iterator
+        pos_map_iterator = ac_map_.find(pos);
+
+    // Return -1.0 if the allele was not called
+    if (pos_map_iterator == ac_map_.end()) {
+      return -1.0;
+    }
+
+    const std::pair<int, std::unordered_map<std::string, int>>& pos_map =
```

Comment:
This is begging for an `auto`.

Reply:
this is actually older code, but I can fix it here